sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/AggregateQueryTests.java

```diff
@@ -7,6 +7,7 @@
 import com.azure.cosmos.CosmosAsyncContainer;
```

Comment:
nit: Unused imports (not related to this).

Reply:
Fixed